docs/spec/v1beta2/helmcharts.md

```diff
@@ -624,3 +624,40 @@ reconcile](#triggering-a-reconcile).
 
 [typical-status-properties]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties
 [kstatus-spec]: https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus
+
+
+## Using The Helm Index
+
+When using a Helm Repository as source for a chart and its dependencies, the source-controller loads the Helm Repository index to find the latest version of a Helm chart.
+
+The source-controller provides a cache for the Helm Repository index. The cache is used to avoid loading the Helm Repository index for every Helm chart reconciliation.
```

Comment:
```suggestion
The controller can be configured to cache the Helm Repository indexes. The cache is used to avoid loading the Helm Repository indexes for every Helm chart reconciliation.
```